### ISSUE TITLE
nvim-web-devicons integration causes lua error in GVim

### ISSUE THREAD

**Issue by mengelbrecht** [mengelbrecht/lightline-bufferline#77]
@yzlnew [reported](https://github.com/mengelbrecht/lightline-bufferline/commit/d72d2bd235cd99f1770c43929ac5229681db173e#commitcomment-44583428) that PR #68 (commit d72d2bd235cd99f1770c43929ac5229681db173e) causes the following error in GVim: `E370: Lua library cannot be loaded.`

**Comment by mengelbrecht** [comment]
@yzlnew I could not reproduce the error you reported using vim/GVim 8.2.
Please provide more information about your setup and a minimal vimrc config if possible.

**Comment by yzlnew** [comment]
```vim
set runtimepath+=$HOME/.vim
set nocompatible

call plug#begin('~/.vim/plugged')
    Plug 'itchyny/lightline.vim'                          " Lightline
    Plug 'mengelbrecht/lightline-bufferline'
call plug#end()

let g:lightline = {}
let g:lightline.component_expand = {
      \  'buffers': 'lightline#bufferline#buffers'
      \  }
let g:lightline.component_type = {
      \  'buffers': 'tabsel',
      \  }
let g:lightline.component_raw = {'buffers': 1}
let g:lightline.tabline = {'left': [['buffers']], 'right': [['close']]}
let g:lightline#bufferline#show_number = 2
let g:lightline#bufferline#enable_devicons = 1
let g:lightline#bufferline#min_buffer_count = 2
let g:lightline#bufferline#unicode_symbols = 0
let g:lightline#bufferline#clickable = 1
```
Platform: Windows 10
Editor: Both vim and gVim 8.2 with patch 1-1738
PS: Working fine on macOS. Weird that it goes into `if has('nvim')` logic on Windows but when I echo `has('nvim')` returns `0`.

**Comment by mengelbrecht** [comment]
Thanks for the info @yzlnew.
I could reproduce the issue with gVim 8.2.1738. It seems to be a bug in vim and is fixed in version [8.2.1910](https://github.com/vim/vim-win32-installer/releases/tag/v8.2.1910) and newer. Here is the corresponding issue in the vim project: https://github.com/vim/vim/issues/7191

**Comment by yzlnew** [comment]
Thanks a lot. Didn't come to my mind that it's a bug from vim itself.
